packages/manager/apps/dedicated/client/app/account/user/users/users.controller.js

```diff
@@ -22,7 +22,12 @@ export default class UserAccountUsersCtrl {
     this.me = coreConfig.getUser();
     this.userIds = [];
     this.users = [];
+    this.groupsArray = [];
+    this.groupIds = [];
```

Comment:
This is the current pattern around this part of the code.
Here's why (from what I figured out): the es-lint requires for the parameters declared in the constructor to be initialized otherwise it throws an error. Then, there same parameters must be initialized again in order for the auto-render of the table to work correctly. We can talk about it if necessary.